src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1415,18 +1417,18 @@ bool OutputStateMachineEngine::s_ParseColorSpec(const std::wstring_view string,
         for (size_t component = 0; component < 3; component++)
         {
             bool foundColor = false;
-            unsigned int* const pValue = &(rguiColorValues[component]);
+            auto value = colorValues.at(component);

```

Comment:
I'm fairly sure this has to be `auto&` for the use you're envisioning; i'd expect this to actually remain 0 because this is a copy

Reply:
ugh yep.